### DISCUSSION TITLE
Notes appended to workflow

### DISCUSSION THREAD

**Discussion by michelevioglio97** [elsa-workflows/elsa-core#4024] [Ideas]
I think it would be useful to have some notepad-style notes appended to the workflow. A note would show the most significant comments that one would like to see corresponding to a specific area of the workflow, like give some more explications about a Fork, or explain why the WF finishes there

**Comment by sfmskywalker** [comment]
Yes, that's a great idea. There's an issue for that [here](https://github.com/elsa-workflows/elsa-core/issues/3736).
